boxsdk/util/chunked_uploader.py

```diff
@@ -31,6 +33,8 @@ def __init__(self, upload_session, content_stream, file_size):
         self._file_size = file_size
         self._part_array = []
         self._sha1 = hashlib.sha1()
+        self._part_definitions = defaultdict(list)
```

Comment:
Why is this a `defaultdict(list)`?  When the key isn't found, this will return an empty list; is that the desired behavior?